Makefile

```diff
@@ -85,24 +98,21 @@ endif
 
 ## Execute config import from config/sync directory
 cim:
-	$(call php, drush cr)
 	$(call php, drush cim -y)
+	$(call php, drush cr)
```

Comment:
wrong change, import already clean caches but to make import you sometimes needs updated codebase